pkgs/games/freeorion/default.nix

```diff
@@ -0,0 +1,23 @@
+{ stdenv, fetchgit, cmake, boost, SDL2, python2, freetype, openal, libogg, libvorbis, zlib, libpng, libtiff, libjpeg, mesa, glew, git, doxygen }:
+
+stdenv.mkDerivation rec {
+  version = "2016-04-16";
+  name = "freeorion-${version}";
+
+  src = fetchgit {
+    url = "https://github.com/freeorion/freeorion";
```

Comment:
First, the patch is not mine. And I'm not actually sure that would fix the performance problem, it's just an educated guess, need to check: https://github.com/freeorion/freeorion/commit/92455f97c28055e296718230d2e3744eccd738ec .

Better to cherry pick the commit into 0.4.5 or to use the git master branch as it is now?